src/utils/case-insensitive-alphabet.ts

```diff
@@ -0,0 +1,12 @@
+import IShow from "./i-show";
+
+export default function caseInsensitiveAlphabet(a: IShow, b: IShow): number {
```

Comment:
Nice function can't complain

Reply:
Could add a test for this function or just a comment explaining what it does for completeness